libdnf/hy-subject.h

```diff
@@ -81,11 +82,12 @@ HyQuery hy_subject_get_best_query(HySubject subject, DnfSack *sack, gboolean wit
 * @param forms HyForm *forms or NULL
 * @param obsoletes If TRUE, obsoletes will be added to result
 * @param reponame Id of repo
+* @param with_src Include source packages
 * @return HySelector
 */
 HySelector hy_subject_get_best_sltr(HySubject subject, DnfSack *sack, HyForm *forms, bool obsoletes,
-                                    const char *reponame);
-HySelector hy_subject_get_best_selector(HySubject subject, DnfSack *sack);
+                                    const char *reponame, bool with_src);
+HySelector hy_subject_get_best_selector(HySubject subject, DnfSack *sack, bool with_src);
```

Comment:
I thing that extra argument here is not necessary, because it should be a default behavior. What about to replace hy_subject_get_best_selector by hy_subject_get_best_sltr. Anyway you propose to break API.

Reply:
Rpm-ostree is only know user of hy_subject_get_best_selector.